lab6/src/components/MusicPlayerControls/NextBtn.jsx

```diff
@@ -0,0 +1,11 @@
+import { ReactComponent as Next } from '../../svgs/Next.svg'
+
+const NextBtn = ({ onClick }) => {
+    return (
+        <button className="controls-button controls-button-next" onClick={onClick}>
+            <Next />
+        </button>
+    )
+}
+
```

Comment:
All ur buttons should be a '<Button/>' but u can manipulate its appearance w/ outer styles and children props, and behaviour with onClick callback